### ISSUE TITLE
Shuffle is True / False?

### ISSUE THREAD

**Issue by 1941302069** [Peiyance/Atten-Mixer-torch#2]
Hello author, I found that the training set did not use shuffle in the GNN-AM code you provided. Why is that? Also, I found that using shuffle = True does not achieve the results proposed in the paper.
